src/analysis/normed/ring/seminorm.lean

```diff
@@ -1,9 +1,12 @@
 /-
 Copyright (c) 2022 María Inés de Frutos-Fernández. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: María Inés de Frutos-Fernández, Yaël Dillies
+Authors: María Inés de Frutos-Fernández, Yaël Dillies, Jiale Miao, Alistair Bill,
+Michał Mrugała, Jan Ot Piña
 -/
+import data.real.basic
```

Comment:
It's really hard to imagine that this import is necessary. Imports are transitive!

Reply:
that's true, I forgot to delete that sorry